src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -38,38 +35,42 @@ public void onAppStart(IApplicationInstance appInstance)
     	}
     }
 
-    
+    // Invoked when an HTTP MPEGDash Streaming session is created; 
+    // defined in IModuleOnHTTPMPEGDashStreamingSession module interfaces
     public void onHTTPMPEGDashStreamingSessionCreate(HTTPStreamerSessionMPEGDash httpSession)
     {
     	getLogger().info("Tommy: SulWowza onHTTPMPEGDashStreamingSessionCreate: " + httpSession.getSessionId());
     	authorizeSession(httpSession);
     }
-    
+
+    // Invoked when an HTTP Cupertino Streaming session is created; 
+    // defined in IModuleOnHTTPCupertinoStreamingSession interface
     public void onHTTPCupertinoStreamingSessionCreate(HTTPStreamerSessionCupertino httpSession)
     {
 		getLogger().info("Tommy: SulWowza onHTTPCupertinoStreamingSessionCreate: ");
 		authorizeSession(httpSession);
 	}
-    
+
     void authorizeSession(IHTTPStreamerSession httpSession)
     {
-    	String query = httpSession.getQueryStr();
-    	if(authorize(query))
-    		httpSession.acceptSession();
-    	else httpSession.rejectSession();
+		String queryStr = httpSession.getQueryStr();
+		if (authorize(queryStr))
+			httpSession.acceptSession();
+		else
+			httpSession.rejectSession();
     }
-    
-    boolean authorize(String query)
+
+    boolean authorize(String queryStr)
     {
-    	String authToken = getAuthToken(query);
-    	return validateAuthToken(authToken);
+		String authToken = getAuthToken(queryStr);
+		return validateAuthToken(authToken);
     }
-    
-    String getAuthToken(String parameters)
+
+    String getAuthToken(String queryStr)
     {
-    	if (parameters != null)
+		if (queryStr != null && queryStr.length() > 6)  // "token=" is 6 chars
     	{
-    		String[] parts = parameters.split("\\?");
+			String[] parts = queryStr.split("\\?");
```

Comment:
The spacing in some of these changes seems really odd.  [Check out the resulting file](https://github.com/sul-dlss/dlss-wowza/blob/tests/src/edu/stanford/dlss/wowza/SulWowza.java#L69-L81)

Would it be worth trying to have consistent indentation in this code?  It kinda seems all over the place right now.